fieldsolver/gridGlue.cpp

```diff
@@ -98,14 +98,14 @@ This is using a 3D, 5-point stencil triangle kernel.
 */
 void filterMoments(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
                            FsGrid< std::array<Real, fsgrids::moments::N_MOMENTS>, FS_STENCIL_WIDTH> & momentsGrid,
-                           FsGrid< fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid) 
+                           FsGrid< fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid)
```

Comment:
Changing argument list indentation like that goes beyond a simple search&replace of the source, but actually needs semantic understanding of the code structures. This is what clang-format is for, and we *can* decide to put it into use for that, but note that this would then also enforce style guides in any other part of the code.

You can't basically make a distinction between "This argument list is oddly formatted because humans find it more readable" and "The absurdly long and convoluted math expressions in ldz_hall.cpp are oddly formatted because that's just how they happen to be".

Reply:
OK. Maybe not now then.